### DISCUSSION TITLE
Localisation support

### DISCUSSION THREAD

**Discussion by jjkislele** [PlayCover/PlayCover#367] [Q&A]
Hi! I'm just doing some localisation jobs. And I found the inconsistent string as below:

https://github.com/PlayCover/PlayCover/blob/6e51e0107e8c9a93c4afd6e22e4e09b65a347b7a/PlayCover/en.lproj/Localizable.strings#L12

The "Activate Account" or the "Switch Account", which one is your expectation?

**Comment by Depal1** [comment]
@IsaacMarovitz

**Comment by amirsaam** [comment]
Does it activate an account?
I mean I don't have Genshin, if an account stays logged in after closing the game and PlayCover (which should be because of the whole process explained in docs), then there is no activation, if you have multiple account, you are literally switching between them not activating any.
So as far as I understand the language, Switch Account is the correct one to use.

**Reply by jjkislele** [reply]
Thanks for your reply. 👍  I think it's reasonable according to your description. But I still want to hear from maintainers, just for double-checking. (In this case, all "Activate Account"-related translations should be changed.)
